pkg/auth/cli.go

```diff
@@ -0,0 +1,45 @@
+package auth
+
+import (
+	"io"
+
+	"github.com/spf13/pflag"
+)
+
+// LoginOptions represents the results for flags in login
+type LoginOptions struct {
+	Authfile      string
+	CertDir       string
```

Comment:
Nothing here implements `Authfile` (probably should be`AuthFile`) and `CertDir`.

If the distinction needs to exist at all, this should at the very least clearly document which options are handled by this subpackage and which need to be handled by the caller.

(And if the resolution to the difficulty of safely handling `--tls-verify` is to give up and let the callers do it, \*shrug\* fine, but somewhere around here it should be at least documented so that callers behave consistently.)